metar/Metar.py

```diff
@@ -286,43 +290,56 @@ def xlate_loc( loc ):
 
 ## Helper functions
 
-def _report_match(handler,match):
+def _report_match(handler, match):
   """Report success or failure of the given handler function. (DEBUG)"""
   if match:
-      print(handler.__name__," matched '"+match+"'")
+      _logger.debug(handler.__name__ + " matched '" + match + "'")
```

Comment:
don't you want to avoid using the string add for logging statements and instead use arguments?
```
_logger.debug("%s matched '%s'", handler.__name__, match)
```
Otherwise there is a small performance hit / opportunity for bugs with the string concatenation?

Reply:
excellent point. i really just starting to get familiar with logging